content/docs/faq-versioning.md

```diff
@@ -1,48 +1,49 @@
 ---
 id: faq-versioning
-title: Versioning Policy
+title: Sürümleme Prensipleri
```

Comment:
Suggestion: I believe `Surum Politikasi`  or with your approach `Surumleme Politikasi` is better for this translation.